elements/pfe-button/demo/pfe-button.js

```diff
@@ -40,7 +40,7 @@ for (const button of document.querySelectorAll('pfe-button')) {
 }
 
 (async function() {
-  const PfeIcon = await customElements.whenDefined('pfe-icon');
+  const PfeIcon = await customElements.get('pfe-icon');
```

Comment:
[whenDefined](https://developer.mozilla.org/en-US/docs/Web/API/CustomElementRegistry/whenDefined) resolves with the class when the tag name is defined